src/py/flwr/client/clientapp/app.py

```diff
@@ -139,6 +139,15 @@ def run_clientapp(  # pylint: disable=R0914
 
                 # Execute ClientApp
                 reply_message = client_app(message=message, context=context)
+
+            except grpc.RpcError as e:
+                # pylint: disable=no-member
+                if e.code() == grpc.StatusCode.PERMISSION_DENIED:
```

Comment:
This `StatusCode` is never returned by the `ClientAppIoServicer`

Reply:
Good catch, removed!